<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainDlg</class>
 <widget class="QDialog" name="MainDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>217</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainDlg</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QLabel" name="label">
     <property name="styleSheet">
      <string notr="true">font: 20pt &quot;文泉驿等宽微米黑&quot;;</string>
     </property>
     <property name="text">
      <string>时间日志记录仪</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="styleSheet">
            <string notr="true">font: 20pt &quot;Sans Serif&quot;;</string>
           </property>
           <property name="text">
            <string>任务描述</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditDesc">
           <property name="minimumSize">
            <size>
             <width>512</width>
             <height>0</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 20pt &quot;Sans Serif&quot;;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="btnStart">
           <property name="styleSheet">
            <string notr="true">font: 20pt &quot;Sans Serif&quot;;</string>
           </property>
           <property name="text">
            <string>开始</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnFinish">
           <property name="styleSheet">
            <string notr="true">font: 20pt &quot;Sans Serif&quot;;</string>
           </property>
           <property name="text">
            <string>完成</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnInterrupt">
           <property name="styleSheet">
            <string notr="true">font: 20pt &quot;Sans Serif&quot;;</string>
           </property>
           <property name="text">
            <string>中断</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="labelTime">
       <property name="styleSheet">
        <string notr="true">font: 22pt &quot;Sans Serif&quot;;</string>
       </property>
       <property name="text">
        <string>2017-01-01 00:00:00</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
